src/modules/articles.js

```diff
@@ -0,0 +1,8 @@
+import axios from "axios";
+
+const getArticles = async () => {
+  let result = await axios.get("/articles");
+  return result.data.articles;
+};
```

Comment:
What I would suggest, is that you wrap the entire functionality in an object. Something like this:
```javascript
const Articles = {
  async index() {
    let result = await axios.get("/articles");
    return result.data.articles;
  }
}
export.default Articles
```
and use it like: 
```javascript
Articles.index().then(response => {
  // whatever you need to do
})
```
(I showcased this method in a demo/session a while back)
It is more readable  IMO